pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
leave hostmount as hostmount... if we need a new SCC for recycling with a special mix of hostmount+runasuser:0+..., create that